Lint error

After use 
```
./gradlew build
```

Error display
```
> Lint found errors in the project; aborting build.
  
  Fix the issues identified by lint, or add the following to your build script to proceed with errors:
  ...
  android {
      lintOptions {
          abortOnError false
      }
  }
  ...
  Errors found:
  
***/node_modules/jail-monkey/android/src/main/java/com/gantix/JailMonkey/MockLocation/MockLocationCheck.java:47: Error: The logging tag can be at most 23 characters, was 26 (Mock location check error ) [LongLogTag]
                          Log.e("Mock location check error ", e.getMessage());
```

--- a/android/src/main/java/com/gantix/JailMonkey/MockLocation/MockLocationCheck.java
+++ b/android/src/main/java/com/gantix/JailMonkey/MockLocation/MockLocationCheck.java
@@ -44,7 +44,7 @@ public class MockLocationCheck {
                             }
                         }
                     } catch (NameNotFoundException e) {
-                        Log.e("Mock location check error ", e.getMessage());
+                        Log.e("Mock location check error", e.getMessage());
                     }
                 }
             }

